index.js

```diff
@@ -7,12 +18,95 @@ function ask(questionText) {
   });
 }
 
-start();
+// starts the pgm - 
+start();   
 
+// function starting the pgm
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  startGameSetup();
+  // hard coding to min & max to begin with 
+    let min=1;    
+    let max=100;  
+    range = ((max - min)+1);  
+
+    console.log("Please think of a number between " + min + " and " + max + " (inclusive), in this " + range + " number range.");
+    let secretNumber = await ask("What is your secret number?\nThis Javascript (JS) program (pgm) won't peek...\n");
+    console.log('You entered: ' + secretNumber);  
+  
+    let pgmTries = 1;
+    let cpuGuessNum = getRandomIntInclusive(min, max);
+
+    console.log('(This JS pgm randomly guesses only once) Is it... ' + cpuGuessNum);   
+    let userRespYorN = await ask('? ');
+  // in case user used lower case - 
+    let userRespYorNUC = userRespYorN.toUpperCase();  
+    
+  // if correctAnswer();  
+    if (userRespYorNUC === 'Y') {
+    console.log('You stated that this was the correct number, Yea!');
+  } else if (userRespYorNUC == 'N') {
+    console.log('You stated it was not correct, or ' + userRespYorNUC + ' , Sorry.');
+  } else {
+    cheatMsg ();
```

Comment:
There is a space between the parenthesis and the end of cheatMsg. This causes the program to crash because it doesn't recognise "cheatMsg ()" as a function.

Reply:
I fixed that - I obviously need help with something (the editor settings, or debugging tools) that will help me find and fix these before submission.